src/daft-image/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+mod counting_writer;
+mod image_buffer;
+mod iters;
+pub mod ops;
+pub use counting_writer::CountingWriter;
+pub use image_buffer::DaftImageBuffer;
+pub mod series;
```

Comment:
How much of these need to be `pub`? Is it not just the `series` module that needs to be the public-facing API?